### PR TITLE
Add guard to user()

### PR DESCRIPTION
**Pull request by lordwhizy**
Hi, 
I just added the $guard to the user() helper. auth()->user() will fail to get the authenticated user when you have multiple guards set up in your laravel project, auth($guard)->user() fixes it.

Cheers.

### CODE REVIEW

File: src/helpers/user.php [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <?php
 
-function user()
+function user($guard = null)
 {
+    if($guard) {
+        return auth($guard)->user();
+    }
+
```
Comment (peter279k) [inline src/helpers/user.php]: How about using `return $guard === null ? auth()->user() : auth($guard)->user();`?

File: src/helpers/user.php [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
     return auth()->user();
```
Comment (m1guelpf) [inline src/helpers/user.php]:
```suggestion
    return is_null($guard) ? auth()->user(): auth($guard)->user();
```

**Comment by calebporzio** [comment]
@lordwhizy - why did you close this?

**Comment by CaddyDz** [comment]
OMG you're like the opposite of taylor otwell, wish you're the one who made Laravel

**Pull request closed without merging**